docs/design/features/dotnet-pgo.md

```diff
@@ -0,0 +1,88 @@
+    --process-name [arg]             The process name within the trace of the process to examine. If this is a multi-process trace, at least one of --pid or --process-name must be specified.
+    --clr-instance-id <arg>          If the process contains multiple .NET runtimes, the instance ID must be specified.
+    --reference <arg>...             If a reference is not located on disk at the same location as used in the process, it may be specified with a --reference parameter
+    --exclude-events-before <arg>    Exclude data from events before specified time. Time is specified as milliseconds from the start of the trace.
+    --exclude-events-after <arg>     Exclude data from events after specified time. Time is specified as milliseconds from the start of the trace.
+    --verbosity <arg>                Adjust verbosity level.
+    --compressed                     Generate compressed mibc
+    -h, --help                       Display this usage message.
+```
+
+`dotnet-pgo create-mibc` is used to create mibc file from a trace. It supports nettrace, etl, netperf, and netperf.zip trace file formats. The `--trace-file` and `--output-file-name` switches must be specified.
+
+If using trace file format that supports multiple process tracing, specify some combination of the `--pid`, `--process-name` and `--clr-instance-id` switches as needed. When run from the command line in situations where these arguments are needed, the valid args will be printed to the console. Usage of the nettrace file format is recommended for users attempting to run this tool as part of a CI system.
+
+The `dotnet-pgo` tool is designed to run on a machine which has the same file system layout as the trace. If this is not the case, `dotnet-pgo` may not be able to find assemblies loaded by the application. In those cases, the `--reference` switch may be used to inform `dotnet-pgo` where to find assemblies that are part of the application.
+
+The `--exclude-events-before` and `--exclude-events-after` switches can be used to control which parts of the execution are optimized.
+
+The mibc file format may be compressed. This will require slightly more time to create and process, but it will produce a smaller binary if the mibc files needs to be distributed to multiple developers. Create a compressed mibc by using the `--compressed` switch.
+
+### `merge` command
+Merge multiple Mibc profile data files into one file.
+
+```
+usage: dotnet-pgo merge --input <arg>... --output-file-name <arg> [--exclude-reference <arg>] [--verbosity <arg>] [--compressed] [-h]
+
+    --input <arg>                    Specify the trace file to be parsed.
+    --output-file-name <arg>         Specify the output filename to be created.
+    --exclude-reference <arg>        Exclude references to the specified assembly from the output.
+    --verbosity <arg>                Adjust verbosity level.
```

Comment:
Add support for `-v` and document that the supported levels are minimal, normal, detailed, and diagnostic.